utils/test_dataimport.py

```diff
@@ -302,13 +302,13 @@ def test_holvi_import_2022_10_format(self):
         data = xls.read()
         res = DataImport.import_holvi(SimpleUploadedFile(name, data))
         self.assertDictEqual(
-            res, {"imported": 8, "exists": 0, "error": 0, "failedrows": []}
+            res, {"imported": 11, "exists": 0, "error": 0, "failedrows": []}
```

Comment:
How come this changes from 8 to 11? Didn't you only add one transaction with a refrence in `additionalInformation` to the test data?

Reply:
I added three more lines in the test data file `utils/holvi-account-test-statement-2022-10.xlsx` (one without either, one with nothing looking like a reference and one with something looking like a reference)